src/types/accounts.ts

```diff
@@ -1,4 +1,10 @@
 export interface AccountInfo {
+  relation: 'parent' | 'peer' | 'child',
```

Comment:
Still think we should use `upstream`, `peer`, and `downstream` instead of `parent` / `child`. Parent makes me think of something that spawns other things. Big connectors don't spawn smaller ones, the smaller ones might be customers of the big one but they're still independent. What we're talking about is much more related to ISPs being "upstream" or "downstream" from one another than something like class inheritance.

Reply:
In ISP terms, you could have an upstream peer, a downstream peer, or a peer which is not strictly either. This could cause some confusion because those terms wouldn't be mutually exclusive.

Peer, Parent, and Child are mutually exclusive, and also describe relationships between people which are easy to conceptualize. In hierarchical descriptions, Parent and Child have easy to grasp meanings, so adding the definition of Peer as a side-to-side link isn't hard to conceptualize.